test/exec.js

```diff
@@ -6,14 +6,20 @@ import test from 'ava';
 
 import shell from '..';
 import utils from './utils/utils';
+import mocks from './utils/mocks';
 
 const CWD = process.cwd();
 const ORIG_EXEC_PATH = shell.config.execPath;
 shell.config.silent = true;
 
+test.beforeEach(() => {
+  mocks.init();
```

Comment:
This line is OK as written. This actually exposes a bug (#862), which I will handle.

Reply:
FYI fix is uploaded (#863). It's OK if this PR lands first.